[canton-15209] Remove agreement text from 3.x (#639)
digital-asset/docs.daml.com: docs/3.0.0/docs/concepts/ledger-model/ledger-structure.rst

To illustrate :doc:`contract keys </daml/reference/contract-keys>`, suppose that the contract key for a `PaintOffer` consists of the reference number and the painter.
So Alice can refer to the `PaintOffer` by its key `(P, P123)`.
To make this explicit, we use the notation `PaintOffer @P A &P123` for contracts, where `@` and `&` mark the parts that belong to a key.

> (The difference between `@` and `&` will be explained in the :ref:`integrity section <da-signatories-maintainers>`.)

Reviewer: ```suggestion
(The difference between `@` and `&` is explained in the :ref:`integrity section <da-signatories-maintainers>`.)
```
